packages/pages-components/src/components/link/link.tsx

```diff
@@ -46,26 +51,26 @@ export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
       if (onClick) {
         onClick(e);
       }
-    };
 
-    const useLinkAsLabel = !children && !link.label;
-    const isObfuscate = !humanInteraction && obfuscate;
-    const obfuscatedStyle: React.CSSProperties = {
-      ...props.style,
-      unicodeBidi: "bidi-override",
-      direction: useLinkAsLabel && isObfuscate ? "rtl" : "ltr",
+      if (isObfuscate) {
+        const encodedLink = currentTarget.href.split("/").at(-1);
+        if (encodedLink) {
+          window.location.href = atob(encodedLink);
+        }
+      }
     };
 
-    const renderedLink = isObfuscate ? reverse(link.link) : link.link;
+    const renderedLink = isObfuscate
+      ? "Obfuscated, set a label or child"
```

Comment:
should this get translated?

Reply:
We don't have translations in any components. But either way, this is more meant for an indicator for the dev to set a label properly. Open to other suggestions though.

Reply:
ah ok. I feel like y'all probably want to remove any strings generally and have them be injectable